### DISCUSSION TITLE
表格版面分析模型，这个模型：ppyolov2_r50vd_dcn_365e_tableBank_word ，有没有预训练模型可以下载

### DISCUSSION THREAD

**Discussion by liuzhipengchd** [PaddlePaddle/PaddleOCR#13419] [Q&A]
ppyolov2_r50vd_dcn_365e_tableBank_word，只有推理模型。
我想问下 ppyolov2_r50vd_dcn_365e_tableBank_word 和 picodet_lcnet_x1_0_fgd_layout_cdla，只识别 表格和文本，选择哪个模型作为预训练的效果好

**Comment by nissansz** [comment]
有好的表格结构模型吗？
